lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -1361,6 +1374,11 @@ SPIRVExtInst *LLVMToSPIRVDbgTran::getSource(const T *DIEntry) {
     constexpr size_t MaxStrSize = MaxNumWords * 4 - 1;
     const size_t NumWords = getSizeInWords(Str);
 
+    if (BM->getDebugInfoEIS() == SPIRVEIS_NonSemantic_Shader_DebugInfo_200 &&
+        Ops.size() == MinOperandCount) {
+      Ops.push_back(getDebugInfoNoneId());
+      Ops.push_back(getDebugInfoNoneId());
+    }
```

Comment:
That is not necessary, but, guess it simplifies translation, right?

Reply:
Yes, it does, but I believe it should be necessary in this case. 
This branch relates to translation of Source code, which is saved in Text operand - I think we should fill previous arguments with DebugInfoNone to avoid confusion in reverse translation (is it Checksum Kind or Text operand)